<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceUserArgsDialog</class>
 <widget class="QDialog" name="DeviceUserArgsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>461</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Device user arguments</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Arguments for hardware</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabHW">
    <attribute name="title">
     <string>Hardware</string>
    </attribute>
    <widget class="QTreeWidget" name="deviceTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>List of available hardware</string>
     </property>
     <column>
      <property name="text">
       <string>HwID</string>
      </property>
      <property name="toolTip">
       <string>Hardware ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Seq</string>
      </property>
      <property name="toolTip">
       <string>Sequence</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="toolTip">
       <string>Description</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="deviceImportLayout">
      <item>
       <widget class="QPushButton" name="importDevice">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add the selected hardware in the list below</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/arrow_down.png</normaloff>:/arrow_down.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="addDeviceHwIDLabel">
        <property name="text">
         <string>HwID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addDeviceHwIDEdit">
        <property name="toolTip">
         <string>Hardware Id of non discoverable device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="addDeviceSeqLabel">
        <property name="text">
         <string>Seq</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addDeviceSeqEdit">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sequence of non discoverable device</string>
        </property>
        <property name="inputMask">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addDevice">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add non discoverable device </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/plusw.png</normaloff>:/plusw.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="argStringLayout">
      <item>
       <widget class="QPushButton" name="deleteArgs">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delete arguments and remove hardware from list above</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/bin.png</normaloff>:/bin.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="argStringLabel">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Args</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="argStringEdit">
        <property name="toolTip">
         <string>Edit user arguments of hardware selected above</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTreeWidget" name="argsTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>List of hardware with user arguments</string>
     </property>
     <column>
      <property name="text">
       <string>ND</string>
      </property>
      <property name="toolTip">
       <string>Non discoverable</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HwID</string>
      </property>
      <property name="toolTip">
       <string>Hardware ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Seq</string>
      </property>
      <property name="toolTip">
       <string>Sequence</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Arg string</string>
      </property>
      <property name="toolTip">
       <string>Argument string</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceUserArgsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceUserArgsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
